test/utility/program_options_tests.hpp

```diff
@@ -0,0 +1,93 @@
+#include "gtest/gtest.h"
```

Comment:
I agree. I just wanted to get something usable up and running. I would like to add more, but I would rather focus on adding necessary features first. The CLI parsing will be the least-likely to fail (since we have well-defined inputs) and will be the least scrutinized.